Funcky.Analyzer/Funcky.Analyzer/SyntaxMatcher.cs

```diff
@@ -0,0 +1,48 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Funcky.Analyzer
+{
+    public class SyntaxMatcher
+    {
+        private const string DefaultValue = "NIL";
```

Comment:
What's up with this odd value?

Reply:
I did not want to throw exceptions which could crash the analyzer, since the tests are working fine now, I replaced it with exceptions: c9afa39527ede5cfecd23e8e205a099e7d548ff8